src/macro.js

```diff
@@ -28,7 +33,9 @@ function graphqlMacro({
   // Case 2: import { loader } from 'graphql.macro'
   loader.forEach(referencePath => {
     referencePath.parentPath.node.arguments.forEach(({ value }) => {
-      const queryPath = path.join(filename, '..', value);
+      const queryPath = value.startsWith('./')
+        ? path.join(filename, '..', value)
+        : resolvePathFromCwd(value);
```

Comment:
This condition does not cover the case when relative path starts with `../`. I would rather use `path.isAbsolute` function to check whether path is absolute.

```
const queryPath = path.isAbsolute(value)
    ? resolvePathFromCwd(value)
    : path.join(filename, '..', value);
```

Reply:
After further testing, I found out that `isAbsolute` is not what we need here because it returns `false` for `foo/bar`. `value.startsWith('./') || value.startsWith('../')` suits better.